frontend/utilities/helpers.ts

```diff
@@ -910,6 +910,12 @@ export const getSoftwareBundleTooltipMarkup = (bundle: string) => {
       `;
 };
 
+export const TAGGED_TEMPLATES = {
```

Comment:
Originally on `HostDetailsPage.tsx` and moved to `utilities/helpers.ts` to be used to pass host id as a target from host details page to edit query to live query